src/fairchem/data/oc/core/slab.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         shift: float | None = None,
         top: bool | None = None,
         oriented_bulk: Structure = None,
-        min_ab: float = 0.8,
+        min_ab: float = 8.0,
```

Comment:
was this was a typo from before?

Reply:
yeah looks like a really old typo. Nothing actually called this directly so not much of a concern.